samples/zigbee/light_switch/Kconfig

```
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NUS_CMD
+	bool "Enable NUS command service"
+	select BT
+	select BT_SMP
+	select BT_PERIPHERAL
+	select BT_GATT_NUS
+	select BT_SETTINGS
+	select SETTINGS
+	help
+	  NUS command service
+
+if NUS_CMD
+
+module = NUS_CMD
+module-str = NUS command
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+endif
+
```

maybe options of the sample could be enclosed in its own menu

```suggestion

menu "Zigbee light switch sample"

config NUS_CMD
	bool "Enable NUS command service"
	select BT
	select BT_SMP
	select BT_PERIPHERAL
	select BT_GATT_NUS
	select BT_SETTINGS
	select SETTINGS
	help
	  NUS command service

if NUS_CMD

module = NUS_CMD
module-str = NUS command
source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"

endif

endmenu #Zigbee light switch sample

```

---

I think the idea was to make the NUS component some sort of a plug-in that any example may include.
Right now it is used only in the switch, so it stays with it.